asyncpg/pool.py

```diff
@@ -446,6 +446,14 @@ async def _initialize(self):
 
                 await asyncio.gather(*connect_tasks)
 
+    @property
+    def closed(self):
```

Comment:
Let's make it a method called `is_closing()` and take into account the `_closing` property too.  This would make `Pool` consistent with the asyncio transport API.

Reply:
Fixed!